### DISCUSSION TITLE
Is it possible to pair Ikea / Philips Hue bulbs with zigbee2mqtt without unpairing them from Ikea / Philips bridges?

### DISCUSSION THREAD

**Discussion by adams-family** [Koenkk/zigbee2mqtt#9588] [General]
The motivation here is simple: bridges like Philips Hue bridge have their own advantages, like ease of use for simple tasks, reliability (don't depend on MQTT, HomeAssistant running, etc), ...

I'd like to keep the vendor hubs (namely Ikea and Philips Hue) for simple tasks and add Zigbee2mqtt for more control.

Is that possible?

**Comment by francisp2** [comment]
A Zigbee device can only be controlled by 1 coordinator. So I would say: no.

**Reply by adams-family** [reply]
I understand. Sad, but in this case there's nothing we can do about it. Thanks for the quick answer!

**Reply by Hedda** [reply]
FYI; the answer is definitely; no. That is a hard limitation set by architecture design in the Zigbee specification by then Zigbee Alliance.

**Reply by pacovi** [reply]
I agree about the fact specs are  clear on Zigbee coordinators. As far as I know, the Zigbee Bridge is not a coordinator but a "simple" zigbee router using the ZLL profile.

https://community.smartthings.com/t/the-technical-reason-why-that-hue-device-is-a-bridge-not-a-hub/103171

Then the question could be frased differently, given that you can Touchlink from the Hue Bridge to make devices join an already existing ZLL network, is there a way to Touchlink the Zigbee2mqtt radio so that it's then in the same zigbee network as the already working Hue Bridge and all the other lamps?

I read that some radios CAN Touchlink in Zigbee2mqtt, as the sender device, the one touchlinking another target device. I was wondering if you could could have the z2mqtt device be the Touchlink target instead...

https://www.zigbee2mqtt.io/guide/usage/touchlink.html

As both IKEA and Hue can Touchlink, then it would be a matter of setting up the network according to your preferences, but you could have the 3 in the same ZLL network for improved network coverage
